my-calculator/public/index.html

```diff
@@ -26,7 +26,7 @@
     -->
```

Comment:
- [ ] I think it's better if we try to change the structure of your project. As it is right now you have two different projects, one nested inside of the other. That, and you even have node_modules pushed to GitHub because the folder outside is missing `.gitignore`. It should be something like this.

```
.github
node_modules
public
src
--> components
--> App.js
--> index.js
.gitignore
package-lock.json
package.json
README.md
.babelrc
.eslintrc.json
```

For reference, this is how it looks right now:
![image](https://user-images.githubusercontent.com/79114233/143046889-37527798-27e3-4ad1-9b7a-ad4d0eac9979.png)

Reply:
- [ ] Make sure that `node_modules` it's removed from the repository, and all other files that are not relevant for your project, as: `reportWebVitals.js`, `App.test.js`, etc.